### PR TITLE
[RHACS] Removed old release notes, added patch release notes

### PR DESCRIPTION
**Pull request by gaurav-nelson**
For https://issues.redhat.com/browse/ROX-18352

- Removed old release notes for 4.0
- Added patch release notes

Preview: https://62208--docspreview.netlify.app/openshift-acs/latest/release_notes/41-release-notes.html

**Cherry-pick**
- `rhacs-docs-4.1`

### CODE REVIEW

File: modules/common-attributes.adoc [hunk 1]
```diff
@@ -53,10 +53,10 @@ endif::[]
 :osp: Red Hat OpenShift
 :olm-first: Operator Lifecycle Manager (OLM)
 :olm: OLM
-:rhacs-version: 4.0.2
+:rhacs-version: 4.1.1
 :ocp-supported-version: 4.10
 :product-title: Red Hat Advanced Cluster Security for Kubernetes
-:product-version: 4.0
+:product-version: 4.1
```
Comment (nalhadef) [inline modules/common-attributes.adoc:59]: Not related to this line. Is a content type needed for this file? It's likely already there.

File: release_notes/41-release-notes.adoc [hunk 2]
```diff
@@ -12,15 +12,16 @@ toc::[]
...
+|`4.1.1` |11 July 2023
 |====
 
 [id="about-this-release_{context}"]
 == About this release
 
 [IMPORTANT]
 ====
-If you are using {ocp} and the Operator for upgrades, see xref:../release_notes/41-release-notes.adoc#kernel-module-collection-method_release-notes-41[Kernel module collection method] for important information and steps you must take before upgrading.
+If you are using {ocp} and the {product-title-short} Operator, and your upgrades are failing, see link:https://access.redhat.com/solutions/7023432[RHACS Operator upgrade to 4.1.0 stuck in Pending state due removal of "KernelModule" collection method] for important information and steps you must take to resolve those issues.
```
Comment (nalhadef) [inline release_notes/41-release-notes.adoc:24]: I know the link text includes the title of another topic, but shouldn't there be something after "due," such as ""to?"

File: release_notes/41-release-notes.adoc [hunk 3]
```diff
@@ -345,26 +346,11 @@ To address this change, you must update the configurations accordingly to ensure
...
-    collection: KernelModule
-----
-to:
-+
-[source,yaml]
-----
-    collection: EBPF
-----
-. Click *Save*.
+If you are using {ocp} and the {product-title-short} Operator, and your upgrades are failing, see link:https://access.redhat.com/solutions/7023432[RHACS Operator upgrade to 4.1.0 stuck in Pending state due removal of "KernelModule" collection method] for important information and steps you must take to resolve those issues.
```
Comment (nalhadef) [inline release_notes/41-release-notes.adoc:353]: See previous comment about adding "to" to linked topic title.

**Comment by nalhadef** [comment]
A few very small comments, Gaurav.

File: modules/common-attributes.adoc (hunk 1 above)
Comment (gaurav-nelson) [reply modules/common-attributes.adoc:59]: Thanks @nalhadef Right now it doesn't have a content type, however I'll check what OCP docs uses for the common attributes file and change it.

**Pull request merged**